admin/src/app/auth.service.ts

```diff
@@ -5,19 +5,20 @@ import { tap } from 'rxjs/operators';
 import { Observable } from 'rxjs';
 
 import { environment } from 'src/environments/environment';
-import { ACCESS_TOKEN_KEY } from './Core/getToken';
+import { authorizationHeader, ACCESS_TOKEN_KEY } from './Core/getToken';
 
 @Injectable({
   providedIn: 'root'
 })
 export class AuthService {
   private readonly apiUrl: string;
+  private readonly jwtHelper: JwtHelperService;
   
   public constructor(
-    private readonly http: HttpClient,
-    private readonly jwtHelper: JwtHelperService
+    private readonly http: HttpClient
   ) { 
     this.apiUrl = environment.api;
+    this.jwtHelper = new JwtHelperService();
```

Comment:
Why?

Reply:
Is used to check if token is expired

Reply:
I mean why did you create it by your own? As I can see, that service was a part of DI